.travis.yml

```diff
@@ -1,13 +1,12 @@
 language: node_js
 node_js:
   - 6
+  - 8
 sudo: required
 dist: trusty
 install:
-  - bundle install
-  - npm install -g grunt-cli
-  - npm install -g bigcommerce/stencil-cli
+  - bundle install --path vendor/bundle
   - npm install
 script:
-  - grunt check
-  - stencil bundle
+  - npx grunt check
+  - npx stencil bundle
```

Comment:
why do we need to run stencil bundle on travis ?

Reply:
Yes, right now we just run it. But it does nothing with the result. Also travis will still success even if it fails. So I am not sure how it helps. Should we just pull it out for now till we figure out how we can test bundles ?

Reply:
I had assumed that stencil bundle would return a non-zero status code if it fails to bundle, and if so, it should fail the Travis script. Is that not true?

Reply:
tbh I have never ever seen it fail on stencil bundle 🤷‍♂️